core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts

```diff
@@ -632,13 +711,92 @@ export class CreateServiceComponent implements OnInit {
     this.servicePatterns = env_oss.servicePatterns;
   }
 
+  validateYAML(){
+    const yamlLint = require('yaml-lint'); 
+    yamlLint.lint(this.deploymentDescriptorText).then(() => {
+      console.log('Valid YAML file.');
```

Comment:
After validating yaml file, not displaying any message - like Valid yaml file or Invalid yaml file?

Reply:
Made this change